pkg/storage/gcp/gcp_test.go

```diff
@@ -48,7 +48,7 @@ func (s *Storage) clear() error {
 
 func getStorage(t testing.TB) *Storage {
 	t.Helper()
-	bucketName := randomBucketName(os.Getenv("DRONE_PULL_REQUEST"))
+	bucketName := randomBucketName(os.Getenv("GA_PULL_REQUEST"))
```

Comment:
nit: Is this really required?

Reply:
My guess is that, should there be a bug, you can actually find the bucket. Otherwise it is always random, and you can never find it.